internal/common/metrics/service_wrapper.go

```diff
@@ -46,44 +46,46 @@ type (
-	scopeNameStartWorkflowExecution           = CadenceMetricsPrefix + "StartWorkflowExecution"
-	scopeNameTerminateWorkflowExecution       = CadenceMetricsPrefix + "TerminateWorkflowExecution"
-	scopeNameResetWorkflowExecution           = CadenceMetricsPrefix + "ResetWorkflowExecution"
-	scopeNameUpdateDomain                     = CadenceMetricsPrefix + "UpdateDomain"
-	scopeNameQueryWorkflow                    = CadenceMetricsPrefix + "QueryWorkflow"
-	scopeNameDescribeTaskList                 = CadenceMetricsPrefix + "DescribeTaskList"
-	scopeNameRespondQueryTaskCompleted        = CadenceMetricsPrefix + "RespondQueryTaskCompleted"
-	scopeNameDescribeWorkflowExecution        = CadenceMetricsPrefix + "DescribeWorkflowExecution"
-	scopeNameResetStickyTaskList              = CadenceMetricsPrefix + "ResetStickyTaskList"
-	scopeNameGetSearchAttributes              = CadenceMetricsPrefix + "GetSearchAttributes"
-	scopeNameListTaskListPartitions           = CadenceMetricsPrefix + "ListTaskListPartitions"
-	scopeNameGetClusterInfo                   = CadenceMetricsPrefix + "GetClusterInfo"
+	scopeNameDeprecateDomain                    = CadenceMetricsPrefix + "DeprecateDomain"
+	scopeNameDescribeDomain                     = CadenceMetricsPrefix + "DescribeDomain"
+	scopeNameListDomains                        = CadenceMetricsPrefix + "ListDomains"
+	scopeNameGetWorkflowExecutionHistory        = CadenceMetricsPrefix + "GetWorkflowExecutionHistory"
+	scopeNameGetWorkflowExecutionRawHistory     = CadenceMetricsPrefix + "GetWorkflowExecutionRawHistory"
+	scopeNamePollForWorkflowExecutionRawHistory = CadenceMetricsPrefix + "PollForWorkflowExecutionRawHistory"
+	scopeNameListClosedWorkflowExecutions       = CadenceMetricsPrefix + "ListClosedWorkflowExecutions"
+	scopeNameListOpenWorkflowExecutions         = CadenceMetricsPrefix + "ListOpenWorkflowExecutions"
+	scopeNameListWorkflowExecutions             = CadenceMetricsPrefix + "ListWorkflowExecutions"
+	scopeNameListArchivedWorkflowExecutions     = CadenceMetricsPrefix + "ListArchviedExecutions"
+	scopeNameScanWorkflowExecutions             = CadenceMetricsPrefix + "ScanWorkflowExecutions"
+	scopeNameCountWorkflowExecutions            = CadenceMetricsPrefix + "CountWorkflowExecutions"
+	scopeNamePollForActivityTask                = CadenceMetricsPrefix + "PollForActivityTask"
+	scopeNamePollForDecisionTask                = CadenceMetricsPrefix + "PollForDecisionTask"
+	scopeNameRecordActivityTaskHeartbeat        = CadenceMetricsPrefix + "RecordActivityTaskHeartbeat"
+	scopeNameRecordActivityTaskHeartbeatByID    = CadenceMetricsPrefix + "RecordActivityTaskHeartbeatByID"
+	scopeNameRegisterDomain                     = CadenceMetricsPrefix + "RegisterDomain"
+	scopeNameRequestCancelWorkflowExecution     = CadenceMetricsPrefix + "RequestCancelWorkflowExecution"
+	scopeNameRespondActivityTaskCanceled        = CadenceMetricsPrefix + "RespondActivityTaskCanceled"
```

Comment:
except for this newline, looks like nothing changed in this file ?can you revert it just to make the diff cleaner